introduction.md

```diff
@@ -4,4 +4,10 @@ This contains something.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?